core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -63,7 +64,7 @@
   private int mMode;
   private String mPersistenceState = "";
   private boolean mMountPoint;
-  private ArrayList<FileBlockInfo> mFileBlockInfos = new ArrayList<>();
+  private Map<Long, FileBlockInfo> mFileBlockInfos = new HashMap<>();
```

Comment:
do we have to update `serialVersionUID` if we change the field, and would there be any backwards compatibility issues?

Reply:
updated.
Since this is not persisted, we should not worry about incompatibility.